downstream/modules/platform/proc-controller-image-pull-policy.adoc

```diff
@@ -0,0 +1,26 @@
+[id="proc_controller-image-pull-policy_{context}"]
+
+= controller-image-pull-policy
+
+
+. Under *Image Pull Policy*, click on the radio button to select *Always*, *Never*, or *IfNotPresent*.
+. Define the *Image Pull Secrets*
```

Comment:
```suggestion
. Complete the following steps to define the *Image Pull Secrets*.
```
Added punctuation. The rest is just a suggestion. If you think this provides too much info, please feel free to ignore.